apps/files/src/components/LocationPicker/MoveSidebarMainContent.vue

```diff
@@ -0,0 +1,22 @@
+<template functional>
```

Comment:
Is this visible, ever? When trying the move operation I had no sidebar and no topbar, thus no help content visible. Are there plans to change that? Otherwise I would consider this obsolete.

Reply:
Was a bug. The sidebar is supposed to be visible. Will be fixed in the next commit.